pages/concepts/protocols/portal-sub-protocols/beacon-chain.md

```diff
@@ -0,0 +1,49 @@
+# Beacon Network
+
+***Note that this network is not yet fully production-ready***
+
+The Beacon Network serves data from the Beacon Chain (Ethereum's consensus protocol) designed to support light clients. 
+Light weight nodes and Portal clients will be able to keep up to date with the Ethereum chain and check the canonicality of their post-merge blocks using data retrieved this network.
```

Comment:
```suggestion
Lightweight nodes and Portal clients will be able to use the data retrieved from this network to determine which blocks have made it on-chain, according to validators. This only applies to post-merge blocks.
```
Assuming most people don't immediately understand what canonical means, here's another option for how to explain it.